apps/web/src/server/applications/case/representations/representation/contact-details/utils/get-titles.js

```diff
@@ -0,0 +1,50 @@
+import { mapTitles } from '../../utils/map-titles.js';
```

Comment:
This looks really like  /representations/representation/address-details/utils/get-titles.js
Is there any chance of re-use?